### PR TITLE
Addressed more SME feedback

### PR DESCRIPTION
**Pull request by evelinec**
- Addressed additional feedback from @chyt

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -28,11 +28,13 @@ Learn how to develop tests for your microservices using the Arquillian Liberty M
...
+http://arquillian.org/[Arquillian] is a testing platform to develop automated functional, integration and acceptance tests for your Java applications.
 
-Arquillian allows different application servers to implement their own embedded, managed and remote containers. This guide will use the Arquillian Liberty Managed container, which is an adaptor to run Arquillian tests with Liberty. It's capable of managing the local Liberty server operations, packaging the tests as a `WAR` to be deployed to the Liberty server and then running the tests on the server.
+Arquillian tests looks similar to the common unit tests. The benefit of Arquillian test is that it sets up the test environment and is able to handle the application server life cycle for you, so you can focus on writing tests. Arquillian supports both JUnit and TestNG as the test runner. This guide will use the JUnit test runner. An Arquillian JUnit test has three core components, `@RunWith(Arquillian.class)`, `@Deployment` and `@Test`. Once you have these three components setup, you'll have everything you need to test your microservices. You will learn more about these components in the test class.
```
Comment (chyt) [inline README.adoc]: Grammar issue - "Arquillian tests looks similar to the common unit tests."

Say "Arquillian tests look similar to common unit tests."

File: README.adoc [hunk 2]
```diff
@@ -120,13 +122,17 @@ The ShrinkWrap test archive's contents are defined in this block, which is going
...
+CDI is used to inject an instance of the `InventoryResource` class into this test class. It is then tested by the `testGetPropertiesFromFunction` method, which validates that the system property for the local and service JVM match. This test case calls the `listContents()` method to get all systems that are stored in this inventory and verifies that one system is found and it is `localhost`.Notice the functional test case does not store any system in the inventory, the `localhost` system is from the endpoint test case that ran prior to this test case. The `@InSequence` Arquillian annotation guarantees the test sequence. The sequences are important for the two test methods, as the results in the first test impacts the second one.
```
Comment (chyt) [inline README.adoc]: Missing a space before the sentence starting with "Notice"

Comment (evelinec) [reply README.adoc]: Fixed

File: README.adoc (hunk 1 above)
Comment (evelinec) [reply README.adoc]: Fixed

**Pull request merged**